src/Integration.Vsix.UnitTests/CFamily/CMake/CompilationDatabaseRequestTests.cs

```diff
@@ -38,23 +38,31 @@ public class CompilationDatabaseRequestTests
     {
         private readonly RequestContext ValidContext = new RequestContext("cpp", Mock.Of<ICFamilyRulesConfig>(), "file.txt", "pchFile.txt", null);
         private readonly CompilationDatabaseEntry ValidDbEntry = new CompilationDatabaseEntry { File = "file.txt", Directory = "c:\\", Command = "a command" };
+        private readonly IReadOnlyDictionary<string, string> ValidEnvVars = new Dictionary<string, string> { { "key1", "value1" } };
 
         [TestMethod]
-        public void Ctor_NullArguments_Throws()
+        public void Ctor_InvalidArguments_Throws()
```

Comment:
Renamed - EnvVars can be null.

Reply:
If they can be null, maybe it's worth to be explicit here and pass null, otherwise it looks like a forgotten case.